### PR TITLE
Topology rules

### PR DESCRIPTION
**Pull request by wangyikewxgm**
### Description of your changes

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Update `sidebar.js` if adding a new page.
- [ ] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

File: docs/reference/topology-rule.md [hunk 1]
```diff
@@ -34,8 +36,73 @@ data:
...
+
+The function of these two `configmap` is totally the same.
 
 In the example above, the parent type is `Cloneset` in group `apps.kruise.io`, his child resource type is `v1/Pod`
-All customized rules specified in these configmaps would be merged with built-in rules and take effect in searching child resources.
\ No newline at end of file
+
+All customized rules specified in these configmaps would be merged with built-in rules and take effect in searching child resources.
+
+## Contained by an addon
```
Comment (wonderflow) [inline docs/reference/topology-rule.md]:
```suggestion
## Built inside with Addon
```

File: docs/reference/topology-rule.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
 ---
-title: Resource Topology
+title: Resource type relationship rules
```
Comment (wonderflow) [inline docs/reference/topology-rule.md]:
```suggestion
title: Resource Relationships
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/topology-rule.md [hunk 3]
```diff
@@ -1,26 +1,30 @@
 ---
-title: 配置拓扑树资源关系规则
----
-
-VelaUX 的资源拓扑图能够展现一个应用所管理的全部资源的拓扑图情况。如下所示：
+title: 配置资源类型关系规则
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/topology-rule.md]:
```suggestion
title: 资源关联规则
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/topology-rule.md [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
-## 原理
+总体上这些`资源类型关系规则`主要会在以下场景中被用到：
 
-Vela 系统内置了一些资源映射规则，这些规则规定了一个类型的资源可能包含了哪些子资源，在展示资源拓扑树的时候，系统会根据这些规则搜寻子资源。
-举例来说，系统规定了一个 [Deployment's](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) 类型的资源的资源只可能是 [ReplicaSet](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/)，所以在展示一个 Deployment 的所有子资源时，Vela 会在这个 Deployment 的 namespace 下列出所有的 ReplicaSet 并过滤掉那些 `ownerReference` 不是指向该 Deployment 的结果。
+- 在 VelaUX 的资源拓扑视图中用来展示资源之间的拓扑关系。下面就是一个资源拓扑图的例子：
 
+![image](../resources/tree.png)
+  
+- 通过 cli 使用 `vela port-forward`, `vela logs`, `vela exec` 以及 `vela status --endpoint` 或者在 VelaUX 上查看应用下实例的日志或访问端口等功能时，用来发现应用下面的 pod 或 service。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/topology-rule.md:14]: 这里是个功能亮点，但是只有一句话，感觉很容易被读者忽略这个功能

Comment (wonderflow) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/topology-rule.md:14]: 我建议分开4个点介绍，用一个具体的例子。

**Review by wonderflow — APPROVED** [review]

**Review by Arhell — COMMENTED** [review]
lgtm

**Review by wonderflow — APPROVED** [review]

**Pull request merged**